compiler/test/input/long_lists.gr

```diff
@@ -20,4 +20,4 @@ loop = (n) => {
   }
 };
 
-loop(45) // <- eats up a lot of heap
+loop(25) // <- eats up a lot of heap
```

Comment:
This was reduced because the test previously asked for 70,000 bytes of memory, but WebAssembly pages are only ~65,000 bytes. The test was reduced to only asking for 20,000 bytes.

Reply:
I suppose we could rework the test runner further to test with multiple pages, but I feel that can be saved for another PR.